Remove "not contains" option

Searching on the vault (Maps, Mods, Online Replays) using the "not contains" search parameter returns an error.

```
java.util.concurrent.CompletionException: org.springframework.web.client.HttpServerErrorException$InternalServerError: 500 Server Error
	at org.springframework.aop.interceptor.AsyncExecutionAspectSupport.lambda$doSubmit$3(AsyncExecutionAspectSupport.java:279)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: org.springframework.web.client.HttpServerErrorException$InternalServerError: 500 Server Error
	at org.springframework.web.client.HttpServerErrorException.create(HttpServerErrorException.java:79)
	at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:99)
	at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:79)
	at com.faforever.client.api.JsonApiErrorHandler.handleError(JsonApiErrorHandler.java:31)
	at org.springframework.security.oauth2.client.http.OAuth2ErrorHandler.handleError(OAuth2ErrorHandler.java:84)
	at org.springframework.web.client.ResponseErrorHandler.handleError(ResponseErrorHandler.java:63)
	at org.springframework.web.client.RestTemplate.handleResponse(RestTemplate.java:777)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:735)
	at org.springframework.security.oauth2.client.OAuth2RestTemplate.doExecute(OAuth2RestTemplate.java:128)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:669)
	at org.springframework.web.client.RestTemplate.getForObject(RestTemplate.java:310)
	at com.faforever.client.api.FafApiAccessorImpl.getPage(FafApiAccessorImpl.java:612)
	at com.faforever.client.api.FafApiAccessorImpl.findModsByQuery(FafApiAccessorImpl.java:404)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343)
	at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:206)
	at com.sun.proxy.$Proxy66.findModsByQuery(Unknown Source)
	at com.faforever.client.remote.FafService.findModsByQuery(FafService.java:415)
	at com.faforever.client.remote.FafService$$FastClassBySpringCGLIB$$6c91c77e.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:218)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:746)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
	at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115)
	at org.springframework.aop.interceptor.AsyncExecutionAspectSupport.lambda$doSubmit$3(AsyncExecutionAspectSupport.java:276)
	... 4 more
```

--- a/src/main/java/com/faforever/client/query/SpecificationController.java
+++ b/src/main/java/com/faforever/client/query/SpecificationController.java
@@ -70,7 +70,6 @@ public class SpecificationController implements Controller<Node> {
 
   private static final Map<ComparisonOperator, String> operatorToI18nKey = ImmutableMap.<ComparisonOperator, String>builder()
       .put(RE, "query.contains")
-      .put(EX, "query.notContains")
       .put(EQ, "query.equals")
       .put(NE, "query.notEquals")
       .put(GT, "query.greaterThan")

--- a/src/main/resources/i18n/messages.properties
+++ b/src/main/resources/i18n/messages.properties
@@ -687,7 +687,6 @@ tournament.gameType=Game type:
 tournament.startingAt=Starting at:
 tournament.completedAt=Completed at:
 game.missionName=Mission
-query.notContains=not contains
 
 game.couldNotJoin=Could not join game with id id ''{0}''.
 discord.disallowJoins=Disallow players joining via Discord

--- a/src/main/resources/i18n/messages_cs.properties
+++ b/src/main/resources/i18n/messages_cs.properties
@@ -622,6 +622,5 @@ settings.appearance.backgoundImage=Obrázek na pozadí
 settings.appearance.useDefaultImage=Použít výchozí obrázek
 settings.appearance.chooseImage=Vybrat obrázek
 settings.general.theme.useNoImage=Žádný obrázek
-query.notContains=neobsahuje
 coop.unknownMission=Neznámá mise
 game.couldNotJoin=Nelze se připojit do hry s id ''{0}''.

--- a/src/main/resources/i18n/messages_fr.properties
+++ b/src/main/resources/i18n/messages_fr.properties
@@ -647,7 +647,6 @@ settings.appearance.backgoundImage=Papier peint
 settings.appearance.useDefaultImage=Image par défaut
 settings.appearance.chooseImage=Choisir l'image
 settings.general.theme.useNoImage=Pas d'image
-query.notContains=ne contient pas
 coop.unknownMission=Mission inconnue
 game.couldNotJoin=Impossible de rejoindre "{0}"
 discord.disallowJoins=Empêcher de rejoindre par discord

--- a/src/main/resources/i18n/messages_tr.properties
+++ b/src/main/resources/i18n/messages_tr.properties
@@ -625,7 +625,6 @@ settings.appearance.backgoundImage=Arkaplan görüntüsü
 settings.appearance.useDefaultImage=Varsayılan resmi kullan
 settings.appearance.chooseImage=Resim seç
 settings.general.theme.useNoImage=Resim yok
-query.notContains=İçermiyor
 coop.unknownMission=Bilinmeyen görev
 game.couldNotJoin=İd'si "{0}" olan oyna katılınamadı.
 discord.disallowJoins=Discord ile bağlananlara izin verme

--- a/src/main/resources/i18n/messages_zh.properties
+++ b/src/main/resources/i18n/messages_zh.properties
@@ -640,7 +640,6 @@ settings.appearance.backgoundImage=背景图像
 settings.appearance.useDefaultImage=使用默认图像
 settings.appearance.chooseImage=选择图像
 settings.general.theme.useNoImage=无图像
-query.notContains=不包括
 coop.unknownMission=未知任务
 game.couldNotJoin=无法加入 ID ID 为"{0}"的游戏。
 discord.disallowJoins=禁止玩家通过 Discord 加入

